pyresample/test/test_slicer.py

```diff
@@ -146,7 +146,7 @@ def test_barely_touching_chunks_intersection(self):
         assert x_slice.start > 0 and x_slice.stop < 100
         assert y_slice.start > 0 and y_slice.stop >= 100
 
-    def test_slicing_an_area_with_infinite_bounds(self):
+    def test_slicing_with_dst_area_with_infinite_edges(self):
```

Comment:
I don't understand the new name of the function. Also the docstring needs to be reworded accordingly.